### ISSUE TITLE
Give the interface some color!

### ISSUE THREAD

**Issue by skillsreactor** [skillsreactor/rpg-learning-example#9]
## User Story
```
AS a player
I WANT to see some color on the output of the game
SO THAT I can more easily distinguish important information and be more immersed into the game!
```

## Acceptance Criteria
```
GIVEN I take an action
WHEN I look at the output in the command line
THEN I see the outcome of my action with colors!
```

I think implementing a npm package like [chalk](https://www.npmjs.com/package/chalk) would be a good starting point here. We will need to re-think some aspect of this upon implementing #7 by possibly giving a "sentiment" to each output line so that other user interfaces we might add in the future can interpret the output in whichever way makes the most sense for it!

**Comment by mdurst365** [comment]
not sure how to assign myself but I started working on this part.

**Comment by skillsreactor** [comment]
Closed by #13
